client/my-sites/checkout/checkout-thank-you/marketplace/use-default-thank-you-footer.tsx

```diff
@@ -3,79 +3,122 @@ import { Button } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
 import { useCallback } from 'react';
 import { useSelector } from 'react-redux';
-import { ThankYouSectionProps } from 'calypso/components/thank-you/types';
+import { ThankYouSectionProps, ThankYouNextStepProps } from 'calypso/components/thank-you/types';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
-export function useDefaultThankYouFoooter( slugs: string[] ): ThankYouSectionProps {
+export function useThankYouFoooter(
+	pluginSlugs: Array< string >,
+	themeSlugs: Array< string >
+): ThankYouSectionProps {
 	const translate = useTranslate();
-	const siteId = useSelector( getSelectedSiteId );
 	const siteSlug = useSelector( getSelectedSiteSlug );
+	const [ hasPlugins, hasThemes ] = [ pluginSlugs, themeSlugs ].map(
+		( slugs ) => slugs.length !== 0
+	);
+
+	const footerSteps: FooterStep[] = [
+		...( hasPlugins
+			? [
+					{
+						key: 'thank_you_footer_support_guides',
+						title: translate( 'Support guides' ),
+						description: translate(
+							'Our guides will show you everything you need to know about plugins.'
+						),
+						link: 'https://wordpress.com/support/plugins/',
+						linkText: translate( 'Plugin Support' ),
+						eventKey: 'calypso_plugin_thank_you_plugin_support_click',
+					},
+			  ]
+			: [] ),
+		...( hasPlugins
+			? [
+					{
+						key: 'thank_you_footer_explore',
+						title: translate( 'Keep growing' ),
+						description: translate(
+							'Take your site to the next level. We have all the solutions to help you.'
+						),
+						link: `/plugins/${ siteSlug }`,
+						linkText: translate( 'Explore plugins' ),
+						eventKey: 'calypso_plugin_thank_you_explore_plugins_click',
+					},
+			  ]
+			: [] ),
+		...( hasThemes
+			? [
+					{
+						key: 'thank_you_footer_forum',
+						title: translate( 'WordPress community' ),
+						description: translate(
+							'Have a question about this theme? Get help from the community.'
+						),
+						link: 'https://wordpress.com/forums/',
+						linkText: translate( 'Visit Forum' ),
+						eventKey: 'calypso_plugin_thank_you_forum_click',
+					},
+			  ]
+			: [] ),
+		{
```

Comment:
I think here there should be a condition similar to 
```
...( ! ( hasPlugins && hasThemes )
	? [
```
because I can see four footers when there are a theme and a plugin
<img width="1059" alt="CleanShot 2023-03-22 at 08 43 20@2x" src="https://user-images.githubusercontent.com/3519124/226833833-a3893834-e1b9-4fe3-a5c1-0d567ac8dcf0.png">

Reply:
+1 to this, I think we'll need to add a condition here

Reply:
I had not pushed this commit yet, but I had done this locally. Fixed on https://github.com/Automattic/wp-calypso/pull/74672/commits/afdb237cde24a7c9c0be582dcfefdb67f130c8fa.

Reply:
In addition to this, the last section will be removed by #74672